.github/workflows/release.yml

```diff
@@ -25,5 +25,11 @@ jobs:
       - name: install packages
         run: pnpm install
 
-      - name: Build packages
-        run: pnpm build
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          publish: pnpm release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
When I tried to register an environment variable with the name "GITHUB_TOKEN", I encountered an error message saying "Variable name cannot start with 'GITHUB_' prefix." Can you change this part to a different name?

Reply:
Probably "GITHUB_TOKEN" does not need to be registered. It should be handled internally.
https://docs.github.com/ja/actions/security-guides/automatic-token-authentication